main.js

```diff
@@ -1,3 +1,17 @@
 import './style.css';
 
-console.log('Hello world');
+function tooltip_enable(event) {
+    var tooltip = document.createElement("span")
+    tooltip.innerHTML = "Hello World!";
+    tooltip.setAttribute("class", "tooltip");
+    event.srcElement.appendChild(tooltip);
+}
+function tooltip_disable(event) {
+    event.srcElement.removeChild(event.srcElement.lastChild);
```

Comment:
If you are trying to find a button with `event.srcElement`, you can get access easier by just `this`